src/get_family_names.py

```diff
@@ -1,15 +1,17 @@
 # coding: utf-8
```

Comment:
As the project is based on Python 3, UTF-8 is the default, no need for this `coding` here ; )